src/core/Renderer.cpp

```diff
@@ -150,6 +150,7 @@ void Renderer::drawScene(Camera *camera, Scene *scene, float currentTime)
 
     drawShadowCasters(shaderProgram, scene);
 
+    drawBloom(emissiveShader, scene, w, h, viewProjectionMatrix);
```

Comment:
yes, otherwise the bloom will be on top of the transparent objects, like the menu. But this is a bit of a problem. If there is a transparant object behind a bloom it will not be viewed correctly. A bit of a moment 22. Requires some thinking